src/notification/examples/notification-drawer.js

```diff
@@ -35,6 +35,7 @@
  * @param {boolean} drawerExpanded Flag if the drawer is expanded (only valid if allowExpand is true). Optional, default: false
  * @param {string}  drawerTitle  Title to display for the drawer (leaving this blank will remove the provided expand capability)
  * @param {object} notificationGroups Collection notification groups to add to the drawer. Alternatively, a single group object can be given if categorization is not used.
+ * @param {string} notificationTrackField Optional field from the notifications to use to track by in the notifications listing ($index used otherwise).
```

Comment:
I don't know if it matters a ton, but perhaps `key` instead of `field`, just for consistency with the lingo Angular uses re: `track by`.  Example:

```
Duplicates in a repeater are not allowed. Use 'track by' expression to specify unique keys. Repeater: {0}, Duplicate key: {1}, Duplicate value: {2}
```
From [the docs](https://docs.angularjs.org/error/ngRepeat/dupes).